### DISCUSSION TITLE
Is there any HTTP API to interface trustwallet in own projects?

### DISCUSSION THREAD

**Discussion by typoworx-de** [trustwallet/wallet-core#1308] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Is there any HTTP API to interface trustwallet in own projects? I intensively red the documentation but did only find code snippets and nothing about how f.e. the trustwallet-cli will communicate to the "backend".

**Describe the solution you'd like**
It would be awesome to have an HTTP API that is able to interface to all coins provided by trustwallet multicoin wallet.

**Describe alternatives you've considered**
May be at least some kind of websocket?

**Comment by optout21** [comment, marked as answer]
Wallet-core contains the cryptoghraphy and low-level blockchain related functionality, but it does not include communication with the blockchain nodes or backend (this is part of the mobile apps).  Wallet-core is fully open source, free to use, but backend/nodes infrastructure is not open to the public.
